src/stories/Header.stories.js

```diff
@@ -27,7 +27,7 @@ export const Level2 = Template.bind({});
 Level2.args = {
     ...Header.defaultProps,
     level: 2,
-    text: 'Heading 2',
+    text: '<p>Heading 2</p>',
```

Comment:
Can we use more "realistic" example from editor.js (in editor we can choose between bold, italic, anchor), for example 
```suggestion
    text: 'What is a <b>CMS</b>',
```